Dockerfile

```diff
@@ -1,42 +1,28 @@
-FROM alpine:3.8 AS stage1
```

Comment:
Does racadm not work on alpine?

Reply:
nm, I see your comment in the PR description.

Reply:
It does not. It installs from debs or rpms and is only supported on specific operating systems, so I stuck with the most recent version of Ubuntu that Dell supports.